### ISSUE TITLE
Data liberation discussion unresolved

### ISSUE THREAD

**Issue by throwaway1037** [BlindMindStudios/StarRuler2-Source#81]
The discussion around data liberation (#79) was unresolved when it was unjustly censored, and I doubt there were plans to uncensor it, as that should have already happened within 6 days of the original censoring.

Please may this discussion run its course?
A lot of users had not finished talking.

**Comment by throwaway1037** [comment]
Essentially I was told by one user to put up and shut up, ie. I, nor any other user, deserves freedom, and I should instead, to paraphrase, "spend about a year shoddily making replacement data instead".

This is silly when we could all enjoy the original data, once liberated, instead of being told to go away and spend lots of time duplicating effort.

**Comment by throwaway1037** [comment]
Bump

**Comment by DaloLorn** [comment]
So... 20 days without a word, and you don't think that qualifies as the discussion having run its course, or the "lot of users" having finished talking? 🤔

**Comment by throwaway1037** [comment]
People might be busy or not know this issue was opened.

**Comment by DaloLorn** [comment]
Anyone watching this repository gets email notifications, and there's a webhook in the OpenSR Discord server that posts all events from the Blind Mind or OpenSR repos in real time. Trust me, *we know.*

**Comment by throwaway1037** [comment]
It depends how people have their notifications set up; I can't rely on people having it set up one specific way, it's their choice how they set it up.

**Comment by throwaway1037** [comment]
@DaloLorn, you actively use Discord and this game, so you obviously don't care about your own freedom very much, which is your own choice, but that doesn't justify stripping others of their freedom.
Freedom is a fundamental and inalienable human right, and with published works, having freedom/liberty means being a free/libre work.

You also ignored my last long message on #79 in which I mentioned your misunderstandings and mistakes surrounding the issue of freedom and copyright, and your potential copyright infringement, so this discussion very much has not run its course.

You've told me, to paraphrase, that I don't deserve freedom and should "spend about a year shoddily making replacement data instead".

In summary, you don't care very much about your own freedom, don't seem to know much about freedom, are possibly actively infringing copyright, and show a total lack of empathy towards people who care about freedom.
No wonder you're ignoring these messages.

It doesn't have to be this way; I am willing to have a friendly discussion, but resolution cannot occur if you refuse to engage with others.
Freedom allows a free community to freely discuss issues, but this cannot happen if the issues are censored.

**Comment by Eeems** [comment]
You say you are willing to have a friendly conversation, yet you just spent most of your last comment antagonizing @DaloLorn.

I believe you need to take a step back, and try to align your messaging and rhetoric with your stated goal of this being friendly. You are currently being extremely antagonistic but hiding it behind "I am willing to have a friendly discussion", or "You just don't care about freedom".

Remember, you catch more flies with honey.

**Comment by DaloLorn** [comment]
I engaged with you up to the point where it became obvious that our views on the subject were irreconcilable, at which point there was no further reason to waste both our time trying to reconcile them. Furthermore, as a non-dev with little to no influence on Jon's decision, I was not required to engage with you *at all*; I chose to do so because I didn't expect Jon to magically change his mind, and I felt that explaining his decision was a gentler way of saying "no" than his ongoing refusal to comment. (In hindsight, it seems it's also been an effective way of shielding him from this patience-depleting mess of a discussion. At least *some* good came of it, I guess.)

If it were up to me, this issue would be closed just like the last one you opened. Circumventing moderation by opening a duplicate/spinoff of a closed thread is a violation of the rules of any other online forum I've ever been on, and there's possibly a case to be made that you're currently violating at least one or both of the following GitHub Community Guidelines (emphasis added):

> - **Bullying and harassment**
> 
>   We do not tolerate bullying or harassment. This means *any habitual badgering* or intimidation targeted at a specific person or group of people. *In general, if your actions are unwanted and you continue to engage in them,* there's a good chance you are headed into bullying or harassment territory.
> 
> - **Disrupting the experience of other users**
> 
>   Being part of a community includes recognizing how your behavior affects others and engaging in meaningful and productive interactions with people and the platform they rely on. *Behaviors such as repeatedly posting off-topic comments, **opening empty or meaningless issues or pull requests, or using any other platform feature in a way that continually disrupts the experience of other users are not allowed.*** While we encourage maintainers to moderate their own projects on an individual basis, GitHub staff may take further restrictive action against accounts that are engaging in these types of behaviors.

However, you haven't yet become quite annoying enough for me to take it up with GitHub's moderators, so I've been comfortable with waiting and seeing how the devs will react (if at all). This is rapidly changing: No longer content with simply using the webhook to flood #os-discussion with your ill-conceived, self-sabotaging attempts at rhetoric, you now see fit to antagonise me personally rather than accept that neither of us has any say in whether your request is granted.

I cannot simply unsubscribe from this conversation, because I cannot prevent the webhook from picking it up without shutting the whole thing down. Make no mistake - if nothing else works, and this comes down to a choice between reporting you or demolishing a valuable community asset, I *will* go through the trouble of dealing with the moderators.

**Comment by throwaway1037** [comment]
I apologise for becoming exasperated.
I've been politely and formally presenting my case, followed by almost a month of waiting, just to be ignored by people actively working *against* freedom.
Moreover, by people who don't understand the issue, are infringing copyright, and are totally against the statement that the freedom of everyone must be respected, which is why I took a lot of time to discuss this issue in the first place.

I'll ignore the bogus part of the message about moderation; it's blindingly obvious this is being threatened as a way to silence dissenting opinion, just as the previous issue was censored.
These so-called "irreconcilable" views really boil down to unwillingness to listen to differing views.

If you ignore the issue of freedom, it won't just disappear.
That's why I'm here to take the time to discuss it, which is impossible if others are not willing to engage.

Moreover, I have presented *tons* of arguments, evidence, and links to articles in favour of freedom, yet all the points raised by the other people who actually bothered to discuss the issue is either unfriendly nonsense like "it's not my priority", "that's fine [if it's proprietary]", "I'm satisfied [with it being proprietary]", "[wanting freedom is a] big fuss", etc. or distractions and tangents like referencing The Good Place, a proprietary video series.

**Comment by sol-oriens** [comment]
I am one of the people who gets notifications and who _knows_ about and is _annoyed_ by your argument. I am also one of the modders who was involved in the "discord war" about open source and "freedom" vs copyrights and I have zero desire to engage with you and start over again. I do, however, have the desire that you stop polluting my mailbox with your nonsense while you should have understood long ago that if _everyone_ is against you, either by mentioning it outright or actually not replying at all to / not supporting your appeal, then maybe you're wrong.

Copyright and intellectual property are a thing and people have the freedom to choose how they share their work. Deal with it. Because the "freedom" you are talking about is here: people who work hard to create content are entitled to choose the rules of how they want it to be consumed. End of story.

Now, if I keep seeing more stuff from you in my mailbox, maybe Dalo won't report you right away, but _I will_.

**Comment by throwaway1037** [comment]
> I am one of the people who gets notifications ... about ... your argument. 

> I do, however, have the desire that you stop polluting my mailbox 

> Now, if I keep seeing more stuff from you in my mailbox, maybe Dalo won't report you right away, but *I will*. 

That's your choice; you can set up your notifications and mail settings however you want, and you don't have to read the discussion if you don't want to.
Complaining about someone else speaking when you know you can easily ignore it, yet choose not to, is ludicrous.
Moreover, actively trying to censor people you don't want to listen to is plainly malicious. 

> I am also one of the modders who was involved in the "discord war" 

Evidently you don't care very much about your own freedom since you use Discord, but that's no reason to justify the denial of granting freedom to other people. 

> "freedom" 

> with your nonsense 

Your scare-quoting and calling agruments in favour of freedom "nonsense" demonstrates once again that you don't care about freedom. 

> about open source 

I do not support ["open source"](https://gnu.org/philosophy/words-to-avoid.html#Open) and I never argued in favour of it.
I support [freedom](https://freedomdefined.org).
Moreover, [open source completely misses the point of freedom.](https://www.gnu.org/philosophy/open-source-misses-the-point.html) 

> "freedom" vs copyrights 

The two are not *necessarily* at odds; asserting copyright and then granting freedom under a copyright license is the very thing that makes a work free.
One could always waive copyright and dedicate a work to the Public Domain, as well, although [this lax approach does not preserve freedom in the way copyleft does](https://www.gnu.org/philosophy/why-copyleft.html). 

> I have zero desire to engage with you and start over again 

That's your prerogative; you're free not to do so.
Nobody is forcing you. 

> while you should have understood long ago that if *everyone* is against you, either by mentioning it outright or actually not replying at all to / not supporting your appeal, then maybe you're wrong. 

This is a classic example of the [*argumentum ad populam* (appeal to popularity) logical fallacy](https://en.m.wikipedia.org/wiki/Argumentum_ad_populum). 

Whilst it's true that the mainstream opinion is against freedom, that is a statement of the problem, not a justification of it. 

There are many communities and projects which value freedom, including [GNU](https://gnu.org), [Parabola](https://parabola.nu), [Freedom Defined](https://freedomdefined.org), [Wikimedia Foundation](https://wikimediafoundation.org), etc.

I have no idea how busy other people are, so I can't just assume no reply means they disagree with me; I'm not a telepath.

> Copyright and intellectual property are a thing 

False.
So-called ["intellectual property"](https://www.gnu.org/philosophy/words-to-avoid.html#IntellectualProperty) is an incoherent propaganda term designed to spread confusion.
To quote Richard Stallman, "Anybody using that term is either confused themselves or trying to confuse you." 

I presume you are part of the former and I don't blame you for using language you've heard other people use, if this is indeed the case and you are not deliberately trying to confuse people. 

> people have the freedom to choose how they share their work. 

One cannot merely prepend the phrase "the freedom to" onto something and expect the end result to be a genuine freedom.
To quote Richard Stallman, "Freedom means having control over your own life."
For example, "the freedom to keep slaves" is not a genuine freedom because oppressing others is unjust and prevents the slaves from controlling their own lives. 

The same goes for "the freedom to choose how they share their work"; this is not a genuine freedom because, if it is not free, the work becomes an instrument of unjust power over its users, which prevents the users from controlling their own lives. 

In reality, choosing to make a work proprietary is [power, not freedom](https://www.gnu.org/philosophy/freedom-or-power.html), and unjust power over others at that. 

> Deal with it. 

> End of story. 

This is needless defeatism.
Do you really think people should just accept injustice and give up trying to make the world a better place? 

> people who work hard to create content are entitled to choose the rules of how they want it to be consumed 

This statement uses three propaganda terms:
- ["Create"](https://www.gnu.org/philosophy/words-to-avoid.html#Creator)
- ["Content"](https://www.gnu.org/philosophy/words-to-avoid.html#Content)
- ["Consume"](https://www.gnu.org/philosophy/words-to-avoid.html#Consume) 

As such, your statement is garbled. 

This statement also sets up the situation as if the authors are hard done by with all their "hard work" and as such deserve the ability to restrict others as a result.
I don't doubt the authors worked hard, because they chose in the first place, but that does not justify oppressing others.

What I presume you are trying to say, translated into using unbiased terminology: "authors control copies of works they publish" is once again a statement of the problem, not a justification of it.
I've already explained why this is unethical. 

> I am one of the people who ...  *knows* about ... your argument. 

As I've demonstrated, you evidently do not. 

--- 

I hope you appreciate the time and effort I took to write this detailed reply, and take the time to understand it.
I hold no hard feeling against anyone here. :) 

At the end of the day, I want this community to thrive in freedom, and I want to take part, but I cannot do so in good conscience whilst the work is proprietary.
Liberating it would be trivial, and I ask that this please be done. 

There need not be arguments, which are only caused by the work not being libre.

**Comment by Zireael07** [comment]
>  and I want to take part, but I cannot do so in good conscience whilst the work is proprietary.

You're already taking part in this community by participating in this discussion (and basically rehashing the same topic again after it was closed)

**Comment by throwaway1037** [comment]
@Zireael07, to clarify, I meant taking part in the community by using the works which are currently proprietary and contributing back to the project.

**Comment by sol-oriens** [comment]
I'm impressed that you took the time to write such a lengthy answer to someone who said they wouldn't engage with you. I'll be honest: I barely read it and won't waste my time commenting on it, except on one point: you are right, I can indeed make you go away from my mailbox by blocking you (which I will do right after posting this). I did, however, waste my time to report you for abuse, as promised, and made my case about you violating community guidelines with your trolling. Support will be judge on whether your "issue" is as legitimate as you pretend it to be, I guess.

**Comment by throwaway1037** [comment]
Viewing presenting the case for freedom as so-called "trolling" and "abuse" just shows how sick our society is at the moment.

This issue is extremely important, the reason I cite most being that this project cannot be packaged in repos for 100% libre systems such as [Parabola](https://parabola.nu), which needlessly blocks entire communities which could enjoy the project and contribute back, thus improving it for everyone else.

**Comment by throwaway1037** [comment]
Replying is also important so that people reading the thread can see rebuttals to each person's posts.

**Comment by DaloLorn** [comment]
> Replying is also important so that people reading the thread can see rebuttals to each person's posts.

Given the... "quality", of your rebuttals... I think you've just made a very good case for why anyone with any sympathy for the free software movement should prefer that we *stop* replying to you. 😂

Unlike Sol, I don't think you're trolling. Trolling is an activity which, by its very nature, requires conscious intent to perform, and while your behavior could otherwise qualify, I think you simply lack the self-awareness needed to realize how intensely annoying you've become since you first opened #79.

This annoyance does not win you any followers. There will be no great triumph of reason, logic and liberty over the barbaric, ignorant hordes trying to keep SR2 under their thumb, because we barbarians live in a completely different world from yours. If we can't even agree with the fundamental premises of each others' arguments, how can we possibly acknowledge those arguments as valid?

**Comment by throwaway1037** [comment]
> There will be no great triumph of reason, logic and liberty over the barbaric, ignorant hordes trying to keep SR2 under their thumb, because we barbarians live in a completely different world from yours.

Then escape the barbaric ignorance and join the free world instead. ;)

> If we can't even agree with the fundamental premises of each others' arguments, how can we possibly acknowledge those arguments as valid?

We don't, that's the whole point; I'm trying to convince you that freedom is important.
Failing to reject anti-freedom points of view and propaganda would be inconsistent with that idea and hypocritical.

Many people unfortunately [fall into this trap](https://www.gnu.org/philosophy/contradictory-support.html) and end up making [ruinous compromises](https://www.gnu.org/philosophy/compromise.html) as a result of the cognitive dissonance this fosters.

If you perceive determination as "annoyance", then that indicates you're probably likely to give up easily, since you don't associate persistence with a strong drive for success.

The very fact that you're still engaging in discussion is a sign that you're somewhat willing to listen to other opinions, and I thank you for taking the time to do so.

If not discussion, or lack thereof, how else do you propose liberation be brought about?

**Comment by lucyllewy** [comment]
We agree. Yes, freedom is important when granted. It is also not possible to guarantee freedom when someone wants to retain control over their work. We have no way of granting the freedom you desire because we don't own the copyright to the assets and the owner of the copyright does not want them to be libre-licensed. That's it. You can keep flogging this dead horse, but it won't change the facts that we are unable, not unwilling, to help you.

**Comment by throwaway1037** [comment]
> We agree. 

> it won't change the facts that we are unable, not unwilling, to help you. 

You could have fooled me. 

> Yes, freedom is important when granted. 

I never stated this.
Freedom is important and must always be granted. 

> we don't own the copyright to the assets and the owner of the copyright 

The following words you used are misleading propaganda terms:
- ["Copyright owner"](https://www.gnu.org/philosophy/words-to-avoid.html#CopyrightOwner)
- ["Assets"](https://www.gnu.org/philosophy/words-to-avoid.html#Assets) 

I don't blame you for using these terms, as it's natural for people to use words they hear others using, but I hope you will consider not using words which imply support for anti-freedom rhetoric in future. 

To translate this into unbiased terminology: you do not *hold* the copyright of the *works* in question. 

> We have no way of granting the freedom you desire 

I never said you did.
Rather, as a community, we can easily rally together to demand the works be liberated. 

> You can keep flogging this dead horse 

We haven't even tried standing together yet; the horse is still galloping. 

To everyone reading, please stand with me and leave a message of support for liberation. 

We can make this happen if we try.

**Comment by DaloLorn** [comment]
> You could have fooled me.

We started out as, and still are, unable. But you're not wrong - over the course of this debate, some of us have also grown unwilling.

**Comment by throwaway1037** [comment]
@DaloLorn how about we put our differences aside and stand together for liberation instead?
A truly passionate list of supporters could convince the proprietors to liberate their works.

**Comment by Zireael07** [comment]
You were told repeatedly: the terms of the license for assets mean it is impossible to "liberate" them as you mean it.

**Comment by throwaway1037** [comment]
The fact that the works are proprietary is the issue.

I'm proposing we stand together as a community and ask the proprietors to liberate their works by releasing them under a libre license.

**Comment by DaloLorn** [comment]
> You were told repeatedly: the terms of the license for assets mean it is impossible to "liberate" them as you mean it.

Teeeechnically, he's been told that Jon doesn't want them to be liberated. There's no legal mechanism preventing him from changing his mind and changing to a less-restrictive license, and only the music is completely beyond Blind Mind's ability to relicense.

Regardless, he's singlehandedly changed my stance from "I guess it'd be nice if Jon did that, but I don't think he will" to "screw you, leave us alone already", so my answer to him is still "no, thanks".

**Comment by throwaway1037** [comment]
@DaloLorn, animosity isn't healthy, especially when we can change it for the better.
As I proposed earlier, why don't we put our differences behind us and stand together as a community in favour of liberation?
